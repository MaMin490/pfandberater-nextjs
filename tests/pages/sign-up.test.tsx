import React from 'react';
import { shallow } from 'enzyme';

import SignUp from '../../pages/sign-up';

describe('SignUp', () => {
  it('should render correctly', () => {
    const component = shallow(<SignUp />);

    expect(component).toMatchSnapshot();
  });
});
