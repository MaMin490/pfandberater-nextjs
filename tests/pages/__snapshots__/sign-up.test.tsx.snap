// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`SignUp should render correctly 1`] = `
<div>
  SignUp
</div>
`;
